commands_to_load/Mod.py

```diff
@@ -82,5 +83,460 @@ def __init__(self, bot):
         if eObj is not False:
             await ctx.send(embed=eObj)
 
+    @commands.command(aliases=['propm'])
+    async def propagatemute(self, ctx):
+        """Ensures all channels have the muted role as part of its permissions.""" 
+        ## Avoid the admin category of channels and rules and announcments channel since nobody can talk in there anyway
+
+        logger.info('[Mod propagatemute()] propagatemute function detected by user ' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod propagatemute()] invoking message deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod propagatemute()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+
+        # Check if muted role is there
+        if not MUTED_ROLE:
+            eObj = await em(ctx, description='Muted role is missing', footer='Command error')
+            if eObj is not False:
+                await ctx.send(embed=eObj, delete_after=5.0)
+            return
+
+        # Get guild
+        channels = ctx.guild.channels
+
+        # Set up the perms overwrite
+        overwrite = discord.PermissionOverwrite()
+        setattr(overwrite, 'send_messages', False)
+        setattr(overwrite, 'manage_messages', False)
+        setattr(overwrite, 'manage_channels', False)
+        setattr(overwrite, 'manage_guild', False)
+        setattr(overwrite, 'manage_nicknames', False)
+        setattr(overwrite, 'manage_roles', False)
+        
+        # Loop through channels and change the perms
+        for channel in channels: 
+            if channel.id not in adminChannels:
+                await channel.set_permissions(MUTED_ROLE, overwrite=overwrite)
+
+        eObj = await em(ctx, description='Muted permissions spread though all channels like herpies. Enjoy :)', footer='Moderator action')

```

Comment:
I dont like this `Muted permissions spread though all channels like herpies.` if you want to put this in the comments of the code, thats more OK with me but making a joke like that might turn some people off of the discord channel which defeats the purpose of trying to be friendly.